packages/babel-helper-transform-fixture-test-runner/src/index.ts

```diff
@@ -371,7 +371,7 @@ async function run(task: Test) {
     }
   }
 
-  if (opts.sourceMaps === true) {
+  if (opts.sourceMaps === true || opts.sourceMap === true) {
```

Comment:
`sourceMaps` vs `sourceMap` is confusing. I would prefer we stick to one of them in the test runner.